devenv/setup.sh

```diff
@@ -59,6 +59,34 @@ devDatasources() {
 		ln -s -f ../../../devenv/datasources.yaml ../conf/provisioning/datasources/dev.yaml
 }
 
+undev() {
+    echo -e "\xE2\x9C\x94 Reverting all dev provisioning"
+
+    # Removing generated dashboard files for bulkDashboard
+    COUNTER=0
+    MAX=400
+    while [ $COUNTER -lt $MAX ]; do
+        rm -f "bulk-dashboards/dashboard${COUNTER}.json"
+        let COUNTER=COUNTER+1
+    done
+
+    # Removing generated dashboard and datasource files for bulkAlertingDashboard
+    COUNTER=0
+    MAX=1000
+    while [ $COUNTER -lt $MAX ]; do
+        rm -f "bulk_alerting_dashboards/alerting_dashboard${COUNTER}.json"
+        let COUNTER=COUNTER+1
+    done
+    rm -f "bulk_alerting_dashboards/bulk_alerting_datasources.yaml"
```

Comment:
I think we can do simpler. Also I think you might have missed some files :slightly_smiling_face: 
```suggestion
    # Removing generated dashboard files from bulk-dashboards
    rm -f bulk-dashboards/dashboard*.json
    rm -f bulk-dashboards/bulk-dashboards.yaml
    echo -e "    \xE2\x9C\x94 Reverting bulk-dashboards provisioning"

    # Removing generated folders from bulk-folders
    rm -rf bulk-folders/Bulk\ Folder*
    rm -f bulk-folders/bulk-folders.yaml
    echo -e "    \xE2\x9C\x94 Reverting bulk-folders provisioning"


    # Removing generated dashboard and datasource files for bulk-alerting-dashboards
    rm -f bulk_alerting_dashboards/alerting_dashboard*.json
    rm -f "bulk_alerting_dashboards/bulk_alerting_dashboards.yaml"
    rm -f "bulk_alerting_dashboards/bulk_alerting_datasources.yaml"
    echo -e "    \xE2\x9C\x94 Reverting bulk-alerting-dashboards provisioning"
```

Reply:
@gamab did you try this out?

Reply:
Yes locally on my linux machine :)